go/stellar/stellarsvc/frontend.go

```diff
@@ -55,13 +59,89 @@ func (s *Server) GetWalletAccountsLocal(ctx context.Context, sessionID int) (acc
 	return accts, nil
 }
 
+func (s *Server) GetAccountAssetsLocal(ctx context.Context, arg stellar1.GetAccountAssetsLocalArg) (assets []stellar1.AccountAssetLocal, err error) {
+	details, err := s.remoter.Details(ctx, arg.AccountID)
+	if err != nil {
+		s.G().Log.CDebugf(ctx, "remote.Details failed for %q: %s", arg.AccountID, err)
+		return nil, err
+	}
+
+	if len(details.Balances) == 0 {
+		// add an empty xlm balance
+		details.Balances = []stellar1.Balance{
+			stellar1.Balance{
+				Amount: "0",
+				Asset:  stellar1.Asset{Type: "native"},
+			},
+		}
+	}
+
+	displayCurrency, err := s.remoter.GetAccountDisplayCurrency(ctx, arg.AccountID)
+	if err != nil {
+		return nil, err
+	}
+	if displayCurrency == "" {
+		displayCurrency = defaultOutsideCurrency
+		s.G().Log.CDebugf(ctx, "Using default display currency %s for account %s", displayCurrency, arg.AccountID)
+	}
+	rate, err := s.remoter.ExchangeRate(ctx, displayCurrency)
+	if err != nil {
+		s.G().Log.CDebugf(ctx, "exchange rate error: %s", err)
+		return nil, err
```

Comment:
Are we ok with this error being fatal to the call? If coinmarketcap goes down I'm not sure it's ok to return the latest known exchange rate for however long it takes us to fix.

Reply:
I feel like the api server should be responsible for this.  The frontend uses these values so prominently that I think it should be fatal at this point.

Reply:
I believe the api server caches the exchange rates for a certain amount of time.  I'm not sure what that timeout is offhand.

If we can't figure out the display currency value, then so many of those frontend interfaces are messed up.

Reply:
There are no designs in zeplin but it's easy to imagine that in place of each live worth display "$66.72 USD" it would show in red "Currency conversion failed" or just nothing.

Reply:
What about building a transaction where you are entering a value in another currency?

Reply:
If the api server currency converter is erroring out then that would not work. But we will have anyway the error pathways to represent that. As the frontend calls something like `BuildTransaction` it will return errors and the user will be presented with a red bar that says "error fetching currency conversion: oh nooo" or something.

Reply:
Yes, I understand that, my point is that if the currency converter is down, the keybase gui wallet is basically down.

HOWEVER, i'll make a change to put an error in Worth and make the WorthCurrency 'ERR' or something.

Reply:
Ok thanks, I think that would be great!
Also fine if you'd rather not, I was arguing individual points